andela/hc-millennium-falcon
# #156577929 Assign priorities to checks and display them on the dashboard by check priority

#### What does this PR do?
Assigns priorities to checks so that they are displayed on the dashboard per the priority assigned.

#### Description of Task to be completed?
A user assigns priorities to checks. There are five priorities i.e. `Lowest`, `Low`, `Normal`, `High` and `Emergency`. Emergency priority checks are listed first and Lowest priority checks are listed last. The default priority for checks is `Normal`. 

<img width="1162" alt="screen shot 2018-05-17 at 13 05 14" src="https://user-images.githubusercontent.com/35847046/40171333-a01cb3be-59d3-11e8-9b8e-2b1a9933c67b.png">

#### How should this be manually tested?
A user clicks on the name of the check and sets the priority, immediately the check will either move up the dashboard or the down depending on the priority set.

#### Any background context you want to provide?
Currently, checks don't have priorities.

#### What are the relevant pivotal tracker stories?

[#156577929](https://www.pivotaltracker.com/story/show/156577929)
